{
  "bug_location": "Unknown",
  "severity": 3,
  "categories": [
    "Unknown"
  ],
  "root_cause": "N/A - Analysis failed",
  "issue_number": 12583,
  "title": "After br log pausing, the global checkpoint is still moving forward"
}